src/screens/ContactScreen/elements/Header.js

```diff
@@ -51,7 +51,7 @@ const Header = ({
         disabled={disabledNextBtn}
         buttonStyle={styles.nextContainer}
         onPress={onPressSub}>
-        <Text style={[styles.text, {color: disabledNextBtn ? COLORS.gray6 : COLORS.holyTosca}]}>
+        <Text style={[styles.text, {color: disabledNextBtn ? COLORS.gray6 : COLORS.holytosca}]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color `COLORS.holytosca` has been changed from `COLORS.holyTosca`. Please ensure that the color name is correct in the COLORS object. If it's a typo, it should be corrected.
```diff
- {color: disabledNextBtn ? COLORS.gray6 : COLORS.holytosca}
+ {color: disabledNextBtn ? COLORS.gray6 : COLORS.holyTosca}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->